src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -82,8 +33,14 @@ const HealthCare = () => {
         <HealthCareCard type="messages" />
         {/* Appointments */}
         <HealthCareCard type="appointments" />
+
         {/* Prescriptions */}
-        <HealthCareCard type="prescriptions" />
+        {canAccessRx && (
+          <Prescriptions
+            prescriptions={prescriptions}
```

Comment:
I see a potential visual issue here. When we are fetching prescriptions, this component will act as though there are no prescriptions yet. Ideally it would show a spinner or something to indicate that it's loading data. I realize that isn't part of the mocks, though.

Reply:
Good point, I was thinking of adding that loader to the HealthCare component after I finish wiring up messages, appointments and prescriptions. I'll want to show the loader if any of that data is being fetched.

Reply:
Sounds good! And makes sense to me. Better than potentially having three loaders, one for each health care block :)